idea_town/frontend/static-src/app/views/scroll-header-view.js

```diff
@@ -0,0 +1,58 @@
+import app from 'ampersand-app';
+
+import BaseView from './base-view';
+import template from '../templates/scroll-header-view';
+const changeHeaderOn = 120;
```

Comment:
Not sure if we have any standards/guidelines on `const` variable naming yet and if we want to go with `CHANGE_HEADER_ON` vs `changeHeaderOn`.

Also, may need a brief code comment explaining what the constant/value is for, for future code travelers.